submissions/kolajoTomike/challenge-array3/question3.js

```diff
@@ -0,0 +1,11 @@
+const getSmallestInteger = (arr) => {
```

Comment:
Nice and simple solution. However, it fails a couple of testcases. 
You could try with `nums = [1]`. The function returns 1, and the expected answer is 2.